src/Traits/FieldTypeTrait.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Opstalent\CrudBundle\Traits;
+
+use Opstalent\CrudBundle\Exception\NotAllowedTypeException;
+use Symfony\Component\Form\FormTypeInterface;
+
+/**
+ * Trait FieldTypeTrait
+ * @author Szymon Kunowski <szymon.kunowski@gmail.com>
+ * @package Opstalent\CrudBundle
+ */
+trait FieldTypeTrait
+{
+    /**
+     * @param string $type
+     * @return bool
+     * @throws NotAllowedTypeException|\Error
+     */
+    protected function isTypeAvailable(string $type): bool
+    {
+        if (is_subclass_of(new $type(), FormTypeInterface::class)) {
```

Comment:
it should be working on strings too, so `is_subclass_of($type, FormTypeInterface::class)`